README.md

```diff
@@ -38,6 +38,7 @@ List of links to the various checkers out there on the web for sites, domains, s
 | http://browsershots.org | Test websites on many browser, platforms and devices. | Free |
 | https://www.browserstack.com/ | Test your website for cross browser compatibility on real browsers. Instant access to multiple desktop and mobile browsers. | Paid |
 | https://www.headspin.io/ | Test your website on thousands of real browsers, mobile device operating systems. | Free trial and paid plans |
+| https://testgrid.io | TestOS brings No-code/Low-code test case authoring and management, Remotely Accessible Infrastructure (browser and device cloud), DevSecOps, and Robotic Test Automation capabilities.. Can be integrated in CI workflows. | Free trial, paid |
```

Comment:
1. It's not alphabetical as per the [Contributing Guidelines](https://github.com/Brunty/awesome-checker-services/blob/master/CONTRIBUTING.md).
2. Also please replace the double `..` after `capabilities`
3. If this site provides browser testing, please link specifically to that page rather than the homepage

Reply:
Made the changes in the readme. Can we consider those now?